src/pages/PageMovieDetails.jsx

```
@@ -1,9 +1,61 @@
+          <img
+            src={`https://image.tmdb.org/t/p/w342/${data.poster_path}`}
+            alt={data.title}
+          />
+          <h1>{data.title}</h1>
+          <p>{data.overview}</p>
+          <p>Release Date: {data.release_date}</p>
+          <p>Rating: {data.vote_average}</p>
+          <p>Runtime: {data.runtime} minutes</p>
+          <p>
+            Genres:{' '}
+            {data.genres && data.genres.map((genre) => genre.name).join(', ')}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)